HackYourFuture-CPH/class24-project
# Feature create update code snippet

# Description
 We created endpoints needed to create and update a snippet in the database.

Fixes # (issue)
No issues.
# How to test?

- To test the POST request:
Send a POST request to http://localhost:3000/api/snippets with a request body.
Make sure you have a snippets collection in your MongoDB database.
If the request is successful, you will receive a response containing the ObjectId of the newly created snippet.
Verify that the snippet is successfully saved in the database.
- To test the PUT request:
Send a PUT request to http://localhost:3000/api/snippets/{id}, where {id} is the snippetId of the snippet you want to update in the database.
In the request body, specify the fields that you want to update.  
Send the PUT request with the updated body.
Verify that the specified fields are successfully updated in the snippet with the corresponding snippetId in the database.
By following these steps, you can test the functionality of the POST and PUT requests for the snippets API.

# Checklist


 - ✅  We  have performed a self-review of our own code.
- ✅  We have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- ✅ We have commented our code, particularly in hard-to-understand areas, 
-  ✅ We have made corresponding changes to the documentation.
- ✅ This PR is ready to be merged and not breaking any other functionality.
